gutenberg/blocks/ts/src/linkedevents/api.ts

```diff
@@ -38,7 +38,7 @@ export class LinkedEventsApi {
    * @return place or null if not found
    */
   public findPlace = async (id: string): Promise<any> => {
-    const result = await this.fetchFromLinkedEvents(`/place/${id}`);
```

Comment:
Base URL is passed directly from settings into the Gutenberg block https://github.com/Metatavu/wordpress-linkedevents/blob/bd8fa6c9db07d3f1518327deb8b8b608c66cc223/gutenberg/blocks/blocks.php#L38 so this is probably a configuration issue and can break existing installations.

You should normalize the URL in fetchFromLinkedEvents method instead